x-pack/plugin/esql/qa/server/multi-node/src/javaRestTest/java/org/elasticsearch/xpack/esql/qa/multi_node/Clusters.java

```diff
@@ -8,15 +8,17 @@
 package org.elasticsearch.xpack.esql.qa.multi_node;
 
 import org.elasticsearch.test.cluster.ElasticsearchCluster;
+import org.elasticsearch.test.cluster.local.LocalClusterConfigProvider;
 import org.elasticsearch.test.cluster.local.distribution.DistributionType;
 
 public class Clusters {
-    public static ElasticsearchCluster testCluster() {
+    public static ElasticsearchCluster testCluster(LocalClusterConfigProvider configProvider) {
```

Comment:
Is there a specific reason we need to make this change in this PR? I've tried to understand, but I think I am missing something.

Reply:
I was pulling other features "up" out of single node tests into the main tests. I thought I'd end up with us having a test for this already. We don't, but I figured the updates to the testing was ok to have along for the ride.